instrumentation/view-click/src/main/kotlin/io/opentelemetry/android/instrumentation/view/click/EventBuilderCreator.kt

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.android.instrumentation.view.click
+import android.view.View
+import io.opentelemetry.android.instrumentation.InstallationContext
+import io.opentelemetry.api.OpenTelemetry
+import io.opentelemetry.api.common.Attributes
+import io.opentelemetry.api.incubator.logs.ExtendedLogRecordBuilder
+import io.opentelemetry.api.incubator.logs.ExtendedLogger
+
+object EventBuilderCreator {
+    private var eventLogger =
+        OpenTelemetry
+            .noop()
+            .logsBridge
+            .loggerBuilder("io.opentelemetry.android.view.click.noop")
+            .build() as ExtendedLogger
+
+    @JvmStatic
+    fun configure(context: InstallationContext) {
+        eventLogger =
+            context.openTelemetry
+                .logsBridge
+                .loggerBuilder("io.opentelemetry.android.view.click")
+                .build() as ExtendedLogger
+    }
+
+    @JvmStatic
+    fun createEvent(name: String): ExtendedLogRecordBuilder =
+        eventLogger
+            .logRecordBuilder()
+            .setEventName(name)
+
+    @JvmStatic
+    fun createViewAttributes(view: View): Attributes {
+        val builder = Attributes.builder()
+        builder.put(viewNameAttr, viewToName(view))
+        builder.put(viewIdAttr, view.id.toLong())
+
+        builder.put(xCoordinateAttr, view.x.toDouble())
+        builder.put(yCoordinateAttr, view.y.toDouble())
```

Comment:
In addition to what we have here, how do we feel about doing something like this:

``
if (view instanceof TextView) {
    setAttribute("view.text", (TextView)view.text)
}
```

Since most clickable things (ie. `ButtonView` etc.) will be subclasses of `TextView`, and this may be more descriptive than the resourceEntry/id strategy in `viewToName`

Reply:
We agreed in SIG that we think it's a good idea but would prefer to defer this to a later enhancement.